seq2science/cli.py

```diff
@@ -277,6 +278,8 @@ def _explain(args, base_dir, workflows_dir, config_path):
 
     rules_used = {"start": f"\nPreprocessing of reads was done automatically with workflow tool "
                            f"seq2science v{seq2science.__version__} (https://doi.org/10.5281/zenodo.3921913)."}
+    rules_used = OrderedDict(rules_used)
+
```

Comment:
Just fyi. Python 3.6+ dictionaries are always insertiond ordered. So this actually does not change anything

Reply:
right, I see the variability of the output is in the DAG. It changes with and without the orderedDict...